src/main/java/io/opentelemetry/semconv/HttpAttributes.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.semconv;
+
+import static io.opentelemetry.api.common.AttributeKey.longKey;
+import static io.opentelemetry.api.common.AttributeKey.stringKey;
+import static io.opentelemetry.semconv.AttributeKeyTemplate.stringArrayKeyTemplate;
+
+import io.opentelemetry.api.common.AttributeKey;
+import java.util.List;
+
+// DO NOT EDIT, this is an Auto-generated file from
+// buildscripts/templates/SemanticAttributes.java.j2
+@SuppressWarnings("unused")
+public final class HttpAttributes {
+
+  /**
+   * HTTP request headers, {@code <key>} being the normalized HTTP Header name (lowercase), the
+   * value being the header values.
+   *
+   * <p>Notes:
+   *
+   * <ul>
+   *   <li>Instrumentations SHOULD require an explicit configuration of which headers are to be
+   *       captured. Including all request headers can be a security risk - explicit configuration
+   *       helps avoid leaking sensitive information. The {@code User-Agent} header is already
+   *       captured in the {@code user_agent.original} attribute. Users MAY explicitly configure
+   *       instrumentations to capture them even though it is not recommended. The attribute value
+   *       MUST consist of either multiple header values as an array of strings or a single-item
+   *       array containing a possibly comma-concatenated string, depending on the way the HTTP
+   *       library provides access to headers.
+   * </ul>
+   */
+  public static final AttributeKeyTemplate<List<String>> HTTP_REQUEST_HEADER =
```

Comment:
Q: do we want attribute and metric definitions to be in the same file or different files?

1. `HttpAttributes.HTTP_REQUEST_HEADER` (or `HttpAttributes.REQUEST_HEADER`) and `HttpMetrics.CLIENT_REQUEST_DURATION`

2. `Http.REQUEST_HEADER_ATTRIBUTE` and `Http.CLIENT_REQUEST_DURATION_METRIC`

Reply:
> 1. `HttpAttributes.HTTP_REQUEST_HEADER`  and `HttpMetrics.HTTP_CLIENT_REQUEST_DURATION`

I like this option 👍